public/style/reply.css

```diff
@@ -1,105 +1,103 @@
 #replyArea {
-    height: 11em;
-    padding-top: 1em;
-    border-top: 1px solid rgba(0, 0, 0, 0.1);
+  height: 11em;
+  padding-top: 1em;
+  border-top: 1px solid rgba(0, 0, 0, 0.1);
 }
 
 .likeContainer {
-    height: 70%;
+  height: 70%;
 }
 
 #icons {
-    display: flex;
-    justify-content: space-between;
-    margin-left: 0.5em;
+  display: flex;
+  justify-content: space-between;
+  margin-left: 0.5em;
 }
 
 #saveSVG {
-    width: 1.3em;
-    margin-right: 1.2em;
+  width: 1.3em;
+  margin-right: 1.2em;
 }
 
 #allLikes {
-    margin: 1em 0em 0em 0.5em;
-    display: flex;
-    flex-direction: column;
+  margin: 1em 0em 0em 0.5em;
+  display: flex;
+  flex-direction: column;
 }
 
 .likeFP {
-    width: 1.5em;
-    height: 1.5em;
-    border: 0em;
+  width: 1.5em;
+  height: 1.5em;
+  border: 0em;
 }
 
-#firstPP {
-    z-index: 1;
+#firstProfilePicture {
+  z-index: 1;
 }
 
-#secondPP {
-    margin-left: -0.5em;
+#secondProfilePicture {
+  margin-left: -0.5em;
 }
 
-
 .likeByWho {
-    display: flex;
+  display: flex;
 }
 
 #likedBy {
```

Comment:
these two classes sound like they would be the same thing

Reply:
change to "likeContentContain